beacon-chain/slasher/detect_blocks.go

```diff
@@ -28,6 +30,10 @@ func (s *Service) detectSlashableBlocks(
 		}
 		if isDoubleProposal(proposedBlocks[i].SigningRoot, existingProposal.SigningRoot) {
 			doubleProposalsTotal.Inc()
+			s.blockSlashingsChan <- &ethpb.ProposerSlashing{
```

Comment:
This isn't right, we should instead just send over a feed called proposerSlashingsFeed